Library/Formula/stlsplit.rb

```diff
@@ -0,0 +1,114 @@
+class Stlsplit < Formula
+  homepage "https://github.com/hroncok/stlsplit"
+  url "https://github.com/hroncok/stlsplit/archive/v1.0.tar.gz"
+  sha256 "778c0f8ee8c47c0d7e8146edd97baea95d61e43db2a278f26823135f351f8262"
+
+  depends_on "premake" => :build
+  depends_on "admesh"
+
+  def install
+    system "premake4", "gmake"
+    system "make"
+    # there is no make install
+    mkdir "#{bin}"
+    mkdir "#{lib}"
+    mkdir "#{include}"
+    cp "build/stlsplit", "#{bin}"
```

Comment:
You can replace all of the ` mkdir `s and ` cp `s with ` bin.install "build/stlsplit" ` and similar.